init.go

```diff
@@ -28,6 +32,7 @@ var Debug *log.Logger
 func init() {
 	flag.StringVar(&stdoutLog, "l", "", "log file for stdout")
 	flag.StringVar(&stderrLog, "e", "", "log file for stderr")
+	flag.StringVar(&debugLog, "d", "", "log file for debug") // only if want to use debugLog
```

Comment:
Use value stored here for debug file name instead of `os.Getenv("DEBUG_LOG")` below.

Reply:
Okay, i had change the code using value stored. Please let me know if there is something to be fix again.